blitz_cli_rest_netconf_gnmi/triggers/shutnoshut.yaml

```diff
@@ -0,0 +1,298 @@
+ShutNoShutBgpNeighborCli:
+  source:
+    pkg: genie.libs.sdk
+    class: triggers.blitz.blitz.Blitz
+  test_sections:
+    - change_bgp_neighbor_remote_as:
+        - configure:
+            device: N9Kv1
+            command: |
+              router bgp 4000
+              neighbor 10.1.1.2
+              remote-as 4001
+        - sleep:
+            sleep_time: 10
+    - verify_bgp_neighbor_inactive:
+        - parse:
+            device: N9Kv1
+            command: show bgp vrf all all summary
+            exclude:
+              - contains_key_value('neighbor', '10.1.1.2').contains_key_value('state', 'established')
+    - revert_bgp_neighbor_remote_as:
+        - configure:
+            device: N9Kv1
+            command: |
+              router bgp 4000
+              neighbor 10.1.1.2
+              remote-as 4000
+        - sleep:
+            sleep_time: 30
+    - verify_bgp_neighbor_active:
+        - parse:
+            device: N9Kv1
+            command: show bgp vrf all all summary
+            include:
+              - contains_key_value('neighbor', '10.1.1.2').contains_key_value('state', 'established')
+
+ShutNoShutBgpNeighborDme:
+  source:
+    pkg: genie.libs.sdk
+    class: triggers.blitz.blitz.Blitz
+  test_sections:
+    - change_bgp_neighbor_remote_as:
+        - api:
+            device: N9Kv1
+            function: nxapi_method_nxapi_rest
+            arguments:
+              alias: rest
+              action: convert
+              input_type: cli
+              commands: |
+                router bgp 4000
+                neighbor 10.1.1.2
+                remote-as 4001
+            save:
+              - variable_name: dme_converted_shut_command
+        - api:
```

Comment:
hmmm....

you have already sent it by cli, so you arnet really testing it was sent with dme... maybe dme failed but because cli worked, the testcase wont fail.

hardcode the dme stuff in here I think.